### DISCUSSION TITLE
Taxonomy/Ontology support

### DISCUSSION THREAD

**Discussion by gpldecha** [cvat-ai/cvat#4183] [Q&A]
Is there support or plan to support taxonomy/ontologies ? That is only next possible attributes are shown based on currently selected attributes. This is supported in labelbox, it does not seem to be the case for cvat. One option would of course to be to flatten the taxonomy tree, but that will generate a lot of classes (all possible paths in the tree).

**Comment by nmanovic** [comment]
@gpldecha , could you please submit a feature request? Also it will be great to have your particular use case in mind when we start thinking about a possible solution. Could you please describe your dataset, labels which you have, typical way to annotate data for your project, etc?

**Reply by gpldecha** [reply]
@nmanovic I will submit later this week a feature request with use case example etc..
